src/Evolution/Systems/CurvedScalarWave/Characteristics.cpp

```diff
@@ -50,8 +50,7 @@ std::array<DataVector, 4> characteristic_speeds(
 
 template <size_t SpatialDim>
 void characteristic_fields(
-    const gsl::not_null<Variables<tmpl::list<
-        Tags::VPsi, Tags::VZero<SpatialDim>, Tags::VPlus, Tags::VMinus>>*>
+    const gsl::not_null<Variables<Tags::char_field_tags<SpatialDim>>*>
```

Comment:
I don't like these free floating typelists. I think they need to either be in the system struct or not exist. It's just another, undocumented, word-of-mouth-type free-floating C++ entity that is nearly impossible to track down. It's actually more reasonable to have the list only locally because then it's actually possible to find. I'd suggest dropping this commit.